app/metrics.yaml

```diff
@@ -2073,6 +2073,19 @@ installation:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+  identifier:
+    send_in_pings:
+      - installation
+    type: string
+    description: |
+      The hashed and salted GAID.
```

Comment:
I'd also add "for a short term installation validation test" so our documentation is clear (even if it's only in here for 1-2 days).